app/views/ontologies/sections/metadata/_ontology_relations_network.html.haml

```diff
@@ -1,5 +1,5 @@
 - unless @ontology_relations_data.nil? || @ontology_relations_data.empty?
-  = render SummarySectionComponent.new(title: 'Ontology relations network') do
+  = render SummarySectionComponent.new(title: 'Ontology relations network', link: "", link_title: "Relation with other ontologies either in <span style='color: #3CB371 !important;'>#{portal_name}</span> or <span style='color:#007bff;'>outside</span>") do
```

Comment:
create a helper for the link_title, instead of writing here

Reply:
done